src/lib/remove-specials/types.ts

```diff
@@ -0,0 +1 @@
+export type IRemoveSpecials = (input: string | number) => string
```

Comment:
Não sei se há necessidade de o input ser do tipo number também, visto que caracteres especiais são apenas string

Reply:
Eu imaginei que, na utilização, poderiam ser inseridos números, e quis garantir que eles não seriam erroneamente removidos pela função. Mas se julgar que devo remover, não tem problema nenhum. Só confirma que eu removo!

Reply:
É que se for inserido numbers, não tem como inserir caracteres especiais. Irei tentar dar um exemplo:
`const number = 1` isso é tipo number, ai não tem como inserir '@' por exemplo, então não precisaria da função.
`const numberAsString = '1'` isso é uma string, tem possibilidade de ser digitado '1@', ai a função seria legal.

Fez sentido o motivo de não precisar ela com numbers?

Reply:
Compreendido! Vou arrumar